libcudacxx/codegen/generators/ld_st.h

```diff
@@ -79,11 +79,18 @@ static inline _CCCL_DEVICE void __cuda_atomic_load_memory_order_dispatch(_Fn &__
 static inline _CCCL_DEVICE void __cuda_atomic_load(
   const _Type* __ptr, _Type& __dst, {3}, __atomic_cuda_operand_{0}{1}, {5}, {7})
 {{
+  static_assert(__cccl_ptx_isa >= 840 && (sizeof(_Type) == 16), "128b ld/st is not supported until PTX ISA version 840");
```

Comment:
`__cccl_ptx_isa >= 840 ` 
SM70 too for ld/st, no?

Reply:
We only support SM70+ so we don't need to guard it for earlier architectures.